Madeline

miseminger opened the pull request.

miseminger committed:
fomite porosity

adjusted general_ontofox.txt to include 'fomite'

added:
-'fomite with increased porosity'
-'fomite with decreased porosity'

miseminger committed:
added SARS-CoV-2 genes

added as a subclass of SARS-CoV-2 for now; this data is all from Rhiannon's vocabulary list.

miseminger committed:
tweaks to seroprevalence subclasses

-changed definitions of 'seroprevalence by location' and 'seroprevalence in asymptomatic population'

-imported 'immune population' and 'infected population'

-made a new subclass of 'infected population': 'asymptomatic population'

miseminger committed:
changed equivalency for ''transmission process by population and PPE usage'

is it incorrect to have "'occurs in' [population]" here, since this class is a subclass of '"has input' some 'susceptible organism'"?

miseminger committed:
imported 'human population'

miseminger committed:
vehicles, + added RQs

-added 'bodily fluid material'
-made various environmental materials subclasses of 'vehicle of infectious agent'
-added "transmission process by population, PPE usage and vehicle of infectious agent"

miseminger committed:
transmission edits

-added "transmission process by population and vehicle of infectious agent"

ddooley reviewed (commented):
I think [outbreak infectious disease identification](http://purl.obolibrary.org/obo/DECISION_0000010) may have as output a taxon ? or infectious agent (which could be a taxon).  I think the disease inheres in people, and that might be an output too, but main one would be to identify the infectious agent ,( which would have one or more associated diseases)

miseminger committed:
care terms

added 'supportive care', 'palliative therapy'

removed 'therapeutic procedure' as it had the same definition as 'treatment'

miseminger committed:
outbreak infectious disease identification

changed output of 'outbreak infectious disease identification' from 'outbreak disease' to 'infectious agent'

also added more RQs

miseminger committed:
imported 'infectivity'

miseminger committed:
imported 'basic reproduction number'

miseminger committed:
imported 'disinfectant'
